[qtinterfaceframework] Build Error, on x64-windows

**Host Environment**
- OS: Windows 11 21H2 22000.556
- Compiler: Visual Studio 2019 Enterprise

**To Reproduce**
Steps to reproduce the behavior:
`vcpkg install qtinterfaceframework:x64-windows`

**Failure logs** 
```bash
❯ vcpkg install qtinterfaceframework:x64-windows
Computing installation plan...
The following packages will be built and installed:
    qtinterfaceframework[core]:x64-windows -> 6.2.3#1
Detecting compiler hash for triplet x64-windows...
-- Automatically setting HTTP(S)_PROXY environment variables to 127.0.0.1:7890
Restored 0 packages from C:\Users\TaylorShi\AppData\Local\vcpkg\archives in 121.3 us. Use --debug to see more details.
Starting package 1/1: qtinterfaceframework:x64-windows
Building package qtinterfaceframework[core]:x64-windows...
-- Using cached C:/v/downloads/qtinterfaceframework-6.2.3.tar.gz
-- Cleaning sources at C:/v/buildtrees/qtinterfaceframework/src/6.2.3-60533e447f.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/v/downloads/qtinterfaceframework-6.2.3.tar.gz
-- Applying patch fix-taglib-search.patch
-- Using source at C:/v/buildtrees/qtinterfaceframework/src/6.2.3-60533e447f.clean
-- Using cached msys-mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst.
-- Using cached msys-mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst.
-- Using msys root at C:/v/downloads/tools/msys2/9a1ec3f33446b195
-- Using cached pypa-get-pip-309a56c5fd94bd1134053a541cb4657a4e47e09d.tar.gz.
-- Cleaning sources at C:/v/buildtrees/qtinterfaceframework/src/7a4e47e09d-f4730c06c6.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/v/downloads/pypa-get-pip-309a56c5fd94bd1134053a541cb4657a4e47e09d.tar.gz
-- Using source at C:/v/buildtrees/qtinterfaceframework/src/7a4e47e09d-f4730c06c6.clean
Collecting pip
ERROR: Exception:
Traceback (most recent call last):
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\cli\base_command.py", line 188, in main
    status = self.run(options, args)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\commands\install.py", line 345, in run
    resolver.resolve(requirement_set)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\legacy_resolve.py", line 196, in resolve
    self._resolve_one(requirement_set, req)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\legacy_resolve.py", line 359, in _resolve_one
    abstract_dist = self._get_abstract_dist_for(req_to_install)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\legacy_resolve.py", line 305, in _get_abstract_dist_for
    abstract_dist = self.preparer.prepare_linked_requirement(
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\operations\prepare.py", line 134, in prepare_linked_requirement
    req.populate_link(finder, upgrade_allowed, require_hashes)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\req\req_install.py", line 211, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\index.py", line 1200, in find_requirement
    candidates = self.find_candidates(
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\index.py", line 1183, in find_candidates
    candidates = self.find_all_candidates(project_name)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\index.py", line 1128, in find_all_candidates
    for page in self._get_pages(url_locations, project_name):
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\index.py", line 1282, in _get_pages
    page = _get_html_page(location, session=self.session)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\index.py", line 234, in _get_html_page
    resp = _get_html_response(url, session=session)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\index.py", line 165, in _get_html_response
    resp = session.get(
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\requests\sessions.py", line 546, in get
    return self.request('GET', url, **kwargs)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_internal\download.py", line 624, in request
    return super(PipSession, self).request(method, url, *args, **kwargs)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\requests\sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\requests\sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\cachecontrol\adapter.py", line 53, in send
    resp = super(CacheControlAdapter, self).send(request, **kw)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\requests\adapters.py", line 412, in send
    conn = self.get_connection(request.url, proxies)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\requests\adapters.py", line 309, in get_connection
    proxy_manager = self.proxy_manager_for(proxy)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\requests\adapters.py", line 193, in proxy_manager_for
    manager = self.proxy_manager[proxy] = proxy_from_url(
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\urllib3\poolmanager.py", line 455, in proxy_from_url
    return ProxyManager(proxy_url=url, **kw)
  File "C:\Users\TAYLOR~1\AppData\Local\Temp\tmpmx23mo6d\pip.zip\pip\_vendor\urllib3\poolmanager.py", line 406, in __init__
    raise ProxySchemeUnknown(proxy.scheme)
pip._vendor.urllib3.exceptions.ProxySchemeUnknown: Not supported proxy scheme None
-- Found external ninja('1.10.2').
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:128 (message):
    Command failed: ninja -v
    Working Directory: C:/v/buildtrees/qtinterfaceframework/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\v\buildtrees\qtinterfaceframework\config-x64-windows-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:415 (vcpkg_execute_required_process)
  installed/x64-windows/share/qtbase/qt_install_submodule.cmake:97 (vcpkg_cmake_configure)
  ports/qtinterfaceframework/portfile.cmake:80 (qt_cmake_configure)
  scripts/ports.cmake:145 (include)


Error: Building package qtinterfaceframework:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `git pull` and `.\vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+qtinterfaceframework
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[qtinterfaceframework]+Build+error
including:
  package: qtinterfaceframework[core]:x64-windows -> 6.2.3#1
    vcpkg-tool version: 2022-03-09-1affd32f93b299d5a907816c328ca3ededb73a7e
    vcpkg-scripts version: 5a8a1dfff 2022-03-09 (27 hours ago)

Additionally, attach any relevant sections from the log files above.
```

[config-x64-windows-out.log](https://github.com/microsoft/vcpkg/files/8228379/config-x64-windows-out.log)
[error-logs-x64-windows.txt](https://github.com/microsoft/vcpkg/files/8228380/error-logs-x64-windows.txt)

**Additional context**

When build qt:x64-windows, this is subbuild

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -1021,7 +1021,7 @@ qt5-base:arm64-windows=fail
 qtwebengine:x64-windows=fail
 
 # upstream bug, see https://github.com/microsoft/vcpkg/issues/23766
-qtinterfaceframework:x64-windows=fail
+qtinterfaceframework:x64-windows=skip
 
 # Skip deprecated Qt module
 # (remove after 1 year or longer due to vcpkg upgrade not handling removed ports correctly)

